examples/tests/Earthfile

```diff
@@ -733,9 +884,27 @@ RUN_EARTHLY:
         if $use_tmpfs; then
             export EARTHLY_TMP_DIR=/tmp/earthly-tmpfs
         fi
-        echo running earthly with $target
-        eval \"/usr/bin/earthly-entrypoint.sh $extra_args $target $post_command\"
-        exit_code=\$?
+        if [ -n \"$pre_command\" ]; then
+            eval \"$pre_command\"
```

Comment:
this was required in order to run the command which changes `/etc/hosts` *before* `/usr/bin/earthly-entrypoint.sh`  invokes the buildkitd process.

Reply:
I wonder if we should add a HOSTS command to add known hosts :thinking: